pkgs/data/themes/plata/default.nix

```diff
@@ -1,10 +1,11 @@
 { stdenv, fetchFromGitLab, autoreconfHook, pkgconfig, parallel
-, sassc, inkscape, libxml2, gnome2, gdk_pixbuf, librsvg, gtk-engine-murrine
+, sassc, inkscape, libxml2, glib, gdk_pixbuf, librsvg, gtk-engine-murrine
 , cinnamonSupport ? true
 , gnomeFlashbackSupport ? true
 , gnomeShellSupport ? true
 , openboxSupport ? true
 , xfceSupport ? true
+, mateSupport ? true, gtk3 ? null, marco ? null
```

Comment:
Is there any advantage in defaulting `gtk3` and `marco` to `null`? If there is none, then it may be better to remove that, simplifying the call to the derivation in `top-level/allPackages.nix`

Reply:
If `mateSupport` defaults to `true` and it really depends on `gtk3` and `marco`, then those maybe should not default to `null`.

Reply:
A similar comment applies to `telegramSupport` and `zip` too.

Reply:
I've removed these defaults, so this will inherit these values from `pkgs`.